### PR TITLE
278 directory and file restructuring for components and pages

### PR DESCRIPTION
**Pull request by GithinjiHans**

### CODE REVIEW

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Container, Row } from 'react-bootstrap';
 
-import { HeaderDashboard } from './HeaderDashboard';
-import { Dashboard } from './Dashboard';
+import { HeaderDashboard } from '../components/dashboard/HeaderDashboard';
```
Comment (Tushar-4781) [inline src/pages/Home.tsx:4]: use path aliases

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{
+}
```
Comment (pulkitvyas08) [inline .vscode/settings.json:2]: Don't include your `.vscode` directory in the project

File: src/components/landingpage/UserChoicePanelRight.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 
 import { ILanguage } from '@src/interfaces';
-import { LanguagesList } from '@components/languagechoices/LanguagesList';
+import { LanguagesList } from '@pages/LanguagesList';
```
Comment (pulkitvyas08) [inline src/components/landingpage/UserChoicePanelRight.tsx:5]: I see a new path alias being used here but the declaration for the same hasn't been done anywhere. Refer to `tsconfig.paths.json`

File: src/pages/AddFeelings.tsx [hunk 4]
```diff
@@ -5,10 +5,10 @@ import { useRecoilValue } from 'recoil';
 
 import { HeaderDashboard } from '@components/dashboard/HeaderDashboard';
 import { darkModeState } from '@store';
-import { AddFeelingsChoices } from './AddFeelingsChoices';
+import { AddFeelingsChoices } from '../components/myfeelingspage/AddFeelingsChoices';
```
Comment (pulkitvyas08) [inline src/pages/AddFeelings.tsx:8]: Use the path alias `@components` here

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]

**Comment by pulkitvyas08** [comment]
@GithinjiHans Please refer to #278 once again and read through the requirements of the directory tree. This PR is not in line with that structure. I would recommend making changes according to that PR before moving forward

File: src/components/landingpage/UserChoicePanelRight.tsx (hunk 3 above)
Comment (GithinjiHans) [reply src/components/landingpage/UserChoicePanelRight.tsx:5]: Done

File: src/pages/Home.tsx (hunk 1 above)
Comment (GithinjiHans) [reply src/pages/Home.tsx]: Done

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]
1. Any directory inside the component page that has an odd mix of components that shouldn't together need to be broken into separate components with their own folder containing `ComponentName.tsx` and `ComponentName.scss`
2. The directories inside the `pages` folder also need to have their own SCSS files

**Review by pulkitvyas08 — APPROVED** [review]
@GithinjiHans I am approving this PR but there are still a ton of problems with the structuring that have not been solved. Still, this is a good effort in that direction but won't close #278

**Review by Tushar-4781 — APPROVED** [review]

**Comment by pulkitvyas08** [comment]
@GithinjiHans Please resolve the conflicts from your side

**Comment by pulkitvyas08** [comment]
@GithinjiHans The PR still has merge conflicts. Resolve them and let me know when it is ready for review

**Comment by GithinjiHans** [comment]
@pulkitvyas08...I've solved the merge conflicts

**Review by pulkitvyas08 — APPROVED** [review]

**Pull request merged**